filament/backend/src/metal/MetalEnums.h

```diff
@@ -435,6 +435,21 @@ inline MTLTextureSwizzleChannels getSwizzleChannels(TextureSwizzle r, TextureSwi
             getSwizzle(a));
 }
 
```

Comment:
(Comment about PR description)
> Manual 💁‍♂️
It was cherrypicked from a later PR

Can you instead give some repro steps of how one can run into the original issue? E.g. a diff to a sample project.